seed/checkUserExistence.js

```diff
@@ -0,0 +1,27 @@
+const User = require('~/models/user')
+const Role = require('~/models/role')
+const {
+  roles: { SUPERADMIN }
+} = require('~/consts/auth')
+const SeedSuperAdmin = require('~/seed/seedSuperAdmin')
+const checkRoleExistence = require('~/seed/checkRoleExistence')
+const logger = require('~/logger/logger')
+
+const checkUserExistence = async () => {
+  try {
+    await checkRoleExistence()
+
+    const foundRole = await Role.findOne({ value: SUPERADMIN }).exec()
+    const isUserExist = await User.exists({ role: foundRole })
+
+    if (isUserExist) {
+      return
+    }
+
+    return SeedSuperAdmin.createSuperAdmin()
```

Comment:
Will the value returned here ever be used? If not, maybe it'd be better to not return different types in one function (line 18 vs line 21)?